src/main/java/com/newcubator/jdbcexcel/ExcelWriter.java

```diff
@@ -36,6 +36,7 @@
 public class ExcelWriter {
 
     public static final int ROWS_IN_MEMORY = 500;
+    public static final int MAX_COLUMN_WIDTH_CHARS = 100;
```

Comment:
I guess you should explain this value (why it is here) in a comment as you did in for your PR.
Could you also explain why this value is 100 and not 256?

Reply:
The value was arbitrarily chosen since i feel 100 characters are more than enough to show in column. The value is not 256 since column width is measured in a different unit than character width (See `calculateColumnWidth` for the conversion between those units (which is also an estimated conversion. Apache uses font measuring for the conversion which we want to prevent for performance reasons)

Im going to add a comment and a test to make it more clear